draft-ietf-rtcweb-jsep.xml

```diff
@@ -513,10 +513,13 @@ setRemote(ANSWER) |                                   |
           <t>Note that gathering phases only gather the candidates
           needed by new/recycled/restarting m= sections; other m=
           sections continue to use their existing candidates. Also,
-          when bundling is active, candidates are only gathered (and
-          exchanged) for the m= sections referenced in BUNDLE-tags, as
-          described in
-          <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation" />.</t>
+          if an m= section is bundled (either by having had
+          bundling offered and accepted or by being
+          marked as bundle-only), then candidates will be gathered
+          and exchanged for that m= section if and only if it is referenced
+          in the appropriate BUNDLE-tag,
```

Comment:
I always misread "bundle-tag" as "bundle-group", and then I need to go look up what a bundle-tag is. I think we could clarify this text with something like:

"candidates will be ... that m= section if and only if its MID is a BUNDLE-tag."